src/main/java/org/dependencytrack/resources/v1/ProjectResource.java

```diff
@@ -804,6 +806,56 @@ public Response deleteProject(
         }
     }
 
+    @POST
+    @Path("/batchDelete")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+            summary = "Deletes a list of projects specified by their UUIDs",
+            description = "<p>Requires permission <strong>PORTFOLIO_MANAGEMENT</strong></p>"
+    )
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "204", description = "Projects removed successfully"),
+            @ApiResponse(responseCode = "207", description = "Access is forbidden to the projects listed in the response"),
+            @ApiResponse(responseCode = "401", description = "Unauthorized")
+    })
+    @PermissionRequired(Permissions.Constants.PORTFOLIO_MANAGEMENT)
+    public Response deleteProjects(List<UUID> uuids) {
+        List<UUID> inaccessibleProjects = new ArrayList<>();
+        try (QueryManager qm = new QueryManager()) {
+            for (Iterator<UUID> it = uuids.iterator(); it.hasNext();) {
+                UUID uuid = it.next();
+                final Project project = qm.getObjectByUuid(Project.class, uuid, Project.FetchGroup.ALL.name());
```

Comment:
What's the reason for using `FetchGroup.ALL` here?

Reply:
Indeed, that doesn't seem to be needed since Project.accessTeams is[ part of the default fetch group](https://github.com/DependencyTrack/dependency-track/blob/4cb2f3a061335561d606dc391e2bb7c67aa95b24/src/main/java/org/dependencytrack/model/Project.java#L280). [ProjectQueryManager.hasAccess](https://github.com/DependencyTrack/dependency-track/blob/4cb2f3a061335561d606dc391e2bb7c67aa95b24/src/main/java/org/dependencytrack/persistence/ProjectQueryManager.java#L960) only needs that property from the project, so I'll change to the getObjectByUuid(Class, UUID) signature instead.